src/mobile/src/main/java/tupperdate/android/onboarding/OnboardingConfirmation.kt

```diff
@@ -0,0 +1,99 @@
+package tupperdate.android.onboarding
+
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.AmbientEmphasisLevels
+import androidx.compose.material.ProvideEmphasis
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.ui.tooling.preview.Preview
+import tupperdate.android.R
+import tupperdate.android.appbars.onlyReturnTopBar
+import tupperdate.android.ui.TupperdateTheme
+import tupperdate.android.ui.TupperdateTypography
+import tupperdate.android.ui.material.BrandedButton
+import tupperdate.android.ui.material.BrandedTextField
+
+@Composable
+fun OnboardingConfirmation(
+    onButtonClick: () -> Unit,
+    onReturnClick: () -> Unit,
```

Comment:
We may want to have some names that better represent the semantic action, such as `onConfirmClick` and `onBackClick`.